src/css/profile/mobile/common/footer.less

```diff
@@ -1,17 +1,16 @@
 .LESSui-footer {
```

Comment:
`.LESSui-footer` is for the 'ui-footer'.
Is it need to change the value here?

Reply:
To apply OneUI to the buttons used in the footer, modification is required.

Reply:
I think .LESSui-footer is used in other components.
Need to check it affect to other components.

Reply:
I'm restoring the code to not modify the .LESSui-footer so it doesn't create a problem.
Change ui-popup-footer to control the properties of .ui-btn directly.